### PR TITLE
Updated NEWS, README and quda_constants file for pending 0.8.0 release.

### PR DESCRIPTION
**Pull request by maddyscientist**
First pass at updating for 0.8.0 release.  Needs more updating of installation instructions once @mathiaswagner has finished updating the wiki pages for cmake.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/lattice/quda/pull/412" target="_blank">ReviewNinja

File: README [hunk 1]
```diff
@@ -93,8 +99,9 @@ application.  The QDP and MILC interfaces can be disabled with the,
 e.g., --disable-milc-interface flag.
 
 The eigen-vector solvers (eigCG and incremental eigCG) require the
-installation of the MAGMA dense linear algebra package.  Supported
-versions are MAGMA 1.5.x and 1.6.x, available from
+installation of the MAGMA dense linear algebra package.  It is
+recommended that MAGMA 1.7.x is used, though versions are 1.5.x and
+1.6.x should work.  MAGMA is available from
 http://icl.cs.utk.edu/magma/index.html.  MAGMA is enabled using the
 configure option --with-magma=MAGMA_PATH.
```
Comment (mathiaswagner) [inline README]: Guess we should add the cmake option here.

Comment (maddyscientist) [inline README]: Yes.  Once you've finished the cmake wiki page we can update this as is needed.  Perhaps we don't need to have so much detail in the README and just refer to the wiki?

**Comment by mathiaswagner** [comment]
I am inclined to keep this pull request open until we are finished with all the adjustments needed - i.e. basically until  I updated the wiki for cmake.

**Comment by maddyscientist** [comment]
Sounds like a plan.

**Comment by mathiaswagner** [comment]
I will keep the hot fix branch alive so we can reuse it for the final update.

**Pull request merged**